proto/evmos/epochs/genesis.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+package evmos.epochs.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/tharsis/evmos/x/epochs/types";
+
+message EpochInfo {
+  string identifier = 1;
+  google.protobuf.Timestamp start_time = 2 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false,
+    (gogoproto.moretags) = "yaml:\"start_time\""
+  ];
+  google.protobuf.Duration duration = 3 [
+    (gogoproto.nullable) = false,
+    (gogoproto.stdduration) = true,
+    (gogoproto.jsontag) = "duration,omitempty",
+    (gogoproto.moretags) = "yaml:\"duration\""
+  ];
+  int64 current_epoch = 4;
+  google.protobuf.Timestamp current_epoch_start_time = 5 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false,
+    (gogoproto.moretags) = "yaml:\"current_epoch_start_time\""
+  ];
+  bool epoch_counting_started = 6;
+  reserved 7;
```

Comment:
@ramacarlucho what do we need reserved for?

Reply:
Im not sure on this one, will check